docs/variants/protectli_v1000-series/openness-score.md

```diff
@@ -0,0 +1,90 @@
+# Dasharo Openness Score
+
+This page contains the [Dasharo Openness
+Score](../../glossary.md#dasharo-openness-score) for Protectli V1000 series
+Dasharo releases. The content of the page is generated with [Dasharo Openness
+Score utility](https://github.com/Dasharo/Openness-Score).
+
+## v0.9.0
+
+Openness Score for protectli_v1210_v0.9.3.rom
```

Comment:
only for v1210? What about V1410?

Reply:
We always publish openness score for one variant for reference only. No need to publish for all variants, since differences would be none or neglectable

Reply:
Shouldn't it be `protectli_v1xxx_v0.9.3.rom` or `protectli_v1x1x_v0.9.3.rom`?

Reply:
No, it always show which binary exactly was evaluated.

Reply:
ok, so let's add just one sentence with an explanation. Something like:

> The results below represent an example openness score for the V1210 platform. However, the same results apply to the entire V1000 series.

Reply:
https://github.com/Dasharo/docs/pull/893/files#diff-68e3fc2518a09f5053cae7caa918e002d5b88892537130e963134958c8ec3a27R10-R12

Reply:
I though up something different in the meantime